source/api/commands/task.md

```diff
@@ -84,7 +84,7 @@ Option | Default | Description
 
 # Examples
 
-## Command
+## Event
```

Comment:
hmm, I am not sure what this second level heading means "Event" (or previous "Command"). Like the text that follows it explains typical use cases, right? And then it explains the return value from the event's callback. So maybe the explanation of return value should be moved into the syntax or rules, then we have the heading `Typical use cases` explaining list of things, then we have actual examples below

Reply:
They sections in Examples typically are sectioned by their API signature, so in this case `cy.task(event)` and `cy.task(event, options)` - there's a heading for `Event` and a heading for `Options`. It seems more obvious for other API signatures, but maybe not as obvious for this one cause 'event' is a weird name. But yeah...